stix_shifter/src/json_to_stix/json_to_stix.py

```diff
@@ -19,15 +19,17 @@ def translate_results(self, data_source, data, options, mapping=None):
         :return: STIX formatted results
         :rtype: str
         """
+
+        self.mapping_json = options['mapping'] if 'mapping' in options else {}
         json_data = json.loads(data)
         data_source = json.loads(data_source)
 
-        if(mapping is None):
+        if(not bool(self.mapping_json)):
```

Comment:
Why use : `if not bool(self.mapping_json):`

instead of :`if not self.mapping_json:`?

Is there an edge case that it covers?